graph/telemetry/istio/istio.go

```diff
@@ -145,6 +146,7 @@ func buildNamespaceTrafficMap(ctx context.Context, namespaceInfo graph.Namespace
 		groupBy := "source_cluster,source_workload_namespace,source_workload,source_canonical_service,source_canonical_revision,destination_cluster,destination_service_namespace,destination_service,destination_service_name,destination_workload_namespace,destination_workload,destination_canonical_service,destination_canonical_revision,request_protocol,response_code,grpc_response_status,response_flags"
 
 		// 0) Incoming: query source telemetry to capture unserviced namespace services' incoming traffic (failed requests that never reach a dest)
+		zl.Debug().Msg("QUERY 1")
```

Comment:
Hi @hhovsepy , You can certainly play around with this but just an FYI that we do have query logging already, it's consolidated in util.PromQuery.  Perhaps we should add an optional queryName param to that call, so it can get incorporated the logging/metrics.

Reply:
Thanks @jshaughn , I used this to figure out most frequently queries to optimize, human readable form, in perf test environment.